deploy/vagrant/tachyon_version.yml

```diff
@@ -4,4 +4,4 @@ Type: Local
 # github repo and commit hash 
 Github: 
   Repo: https://github.com/amplab/tachyon
-  Hash: eaeaccd19832494c5f0b2a792888a5eaacd7bcf8
+  Hash: 74f2968d99a9bb5f9ed973c1a226eee25d33ba65
```

Comment:
Not sure if it makes sense to keep updating this hash, maybe just leave it as something the user needs to update (since it very rarely will be what they want).